CIP-XXXX/README.md

```diff
@@ -0,0 +1,312 @@
+
+## Specification
+
+Oracles are responsible for providing multiple types of off-chain data to the blockchain. This proposal outlines an on-chain datum standard for Oracle producers to submit to the blockchain and Oracle consumers to read from the blockchain in a standardized format.
+
+Such a standard should accomplish a number of somewhat contradictory objectives
+
+* Data provided by the feed should be possible to alter over time, to add more detailed data or remove parts of the data, without major disruption to existing users of the data feed
+* Data provided in the feed should be stored in an efficient manner, to avoid data duplication and excessive transaction costs
+* The standard should be flexible enough to capture the majority of use-cases, even those that may not yet be conceived of at the writing of this document
+* The standard should provide room for individual data providers to add their own data fields to a feed, without interfering with the common standard
+
+### Data Format
+
+The basis for this standard is the Concise Binary Object Representation (CBOR) data format, for which more information can be found here: [https://www.rfc-editor.org/rfc/rfc8949.html](https://www.rfc-editor.org/rfc/rfc8949.html) 
+
+A central aspect of the CBOR standard that is used extensively is the concept of a Tag, which allows data consumers to identify and consume the types of data that they expect, and gracefully ignore other pieces of data that they are not interested in.
+
+Similar to the CBOR standard this CIP will also propose a CBOR Tags Registry. Data providers are able to provide data according to the formats provided in this registry, and data consumers can discern the different available formats to pick the exact kind of data they need out of a feed.
+
+To maintain compatibility with the main CBOR Tags Registry, the tags proposed in this are defined as an offset from Base Oracle Datum CIP Tag Number, which is defined later in this document. For example, Tag +5 would correspond to the actual number OracleDatumCipTagNumber + 5.
+
+
+#### Detailed Format
+
+Any data feed datum matching this standard must adhere to the following overall structure. Each of these individual elements is a Tag defined further later on.
+
+The data structure consists of a list of Tags. This list is itself surrounded by a tag versioned to this major version of CIP. A later CIP may thus build on top of this standard by adding further datum after the initial one, using a different tag that allows older users of this CIP to disregard this new data.
+
+There are 3 major categories of Tags, described in further detail later on, SharedData, GenericData and ExtendedData. In the list the ordering is defined as [ ? SharedData, 1* GenericData, ? ExtendedData ]. Or in plain english, a SharedData may optionally come first, then 1 or more GenericData in no particular order, and finally an ExtendedData may optionally come last.
+
+
+### CIP CBOR Tags Registry
```

Comment:
Orcfax were invited to participate in the process, we were very open, but there didn't seem to be a great deal of interest at the time.

Reply:
I'm not a representative of Orcfax. As an MLabs dev/manager, I collaborated with Orcfax on a Catalyst-funded project to implement the [Cardano Open Oracle Protocol (COOP)](https://www.orcfax.link/introducing-the-cardano-open-oracle-protocol-coop/).

It is unfortunate that there weren't more active discussion earlier in your CIP process to potentially converge on a single standard. My comment here merely states the fact that your CIP is incompatible with COOP because you identify oracle providers via integer ID pointers to an external registry, whereas COOP identifies oracle providers via the minting policy ID of the tokens the oracle uses to authenticate its published data. The two approaches are fundamentally not compatible (unless you allow oracle providers to be identified by either an integer ID or a minting policy ID).

Reply:
The two approaches can certainly co-exist in the Cardano ecosystem, and I'm curious to see which one will be more beneficial in practice.

Reply:
Please believe that I am making my comments in good faith and hoping that they are constructive for you. :pray: